app/features/syllabus/prompts/syllabus_prompt.txt

```diff
@@ -0,0 +1,16 @@
+You are AI teacher assistant. You job is to create detailed course syllabus, based on some info provided. Read some notes about the course delimited by XML tags:
+<notes>
```

Comment:
You don't need to include XML tags in the prompt since the input should be in JSON format. LLMs usually understand the start and end of instructions effectively, so Markdown format is generally preferred.